pyphare/pyphare/pharein/__init__.py

```diff
@@ -195,6 +196,7 @@ def as_paths(rb):
         name_path = type_path + diag.name
         add(name_path + "/" + 'type' , diag.type)
         add(name_path + "/" + 'quantity' , diag.quantity)
+        add_size_t(name_path + "/" + "flush_every", diag.flush_every)
```

Comment:
we have to be cautious that "flush every" could be interpreted as "every N time step" although it really means (if I understand correctly) "flush every N dump"